src/Illuminate/Foundation/Console/RouteListCommand.php

```diff
@@ -187,16 +189,31 @@ protected function getMiddleware($route)
     }
 
     /**
-     * Filter the route by URI and / or name.
+     * Filter the route according to options given.
      *
      * @param  array  $route
-     * @return array|null
+     * @return array|void
```

Comment:
Incorrect phpdoc. Needs to be null.

Reply:
I can remove that if you like; I made that change because it was tripping static analysis tools.